Code/ForceField/MMFF/Params.h

```diff
@@ -178,9 +172,8 @@ class RDKIT_FORCEFIELD_EXPORT MMFFDefCollection {
   */
   const MMFFDef *operator()(const unsigned int atomType) const {
 #ifdef RDKIT_MMFF_PARAMS_USE_STD_MAP
-    std::map<const unsigned int, MMFFDef>::const_iterator res;
-    res = d_params.find(atomType);
-
+    std::map<const unsigned int, MMFFDef>::const_iterator res =
```

Comment:
I think this can be an auto:
```suggestion
    const auto res =
```